com.pinegrow.Pinegrow.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Pinegrow
+GenericName=Website Builder for professionals
+Exec=pinegrow --enable-features=UseOzonePlatform --ozone-platform-hint=auto
```

Comment:
these arguments should be in the wrapper script instead.

Reply:
but then usually electron + wayland is awfully buggy.

Reply:
> but then usually electron + wayland is awfully buggy.

In my experience the performances are way better using Wayland with Pinegrow. Also it falls back to X11 so users can disable wayland if they want

Reply:
> these arguments should be in the wrapper script instead.

I'll add that